services/app-api/utils/validation/completionStatus.ts

```diff
@@ -210,6 +211,36 @@ export const calculateCompletionStatus = async (
             route.entityType
           ),
         };
+        // handle Analysis Methods case: this section allows users to add custom methods, which use different form questions
+        if (
+          route.path ===
+            "/naaar/state-and-program-information/analysis-methods" &&
```

Comment:
maybe make this url a constant and refer to it here?

Reply:
@britt-mo the other routes in this switch statement are hardcoded so I will leave it for now, but good callout!